src/main/scala/com/cognite/sdk/scala/v1/fdm/instances/InstancePropertyValue.scala

```diff
@@ -201,10 +184,22 @@ object InstancePropertyValue {
                     InstancePropertyValue.Float64List(arr.flatMap(_.asNumber).map(_.toDouble))
                 }
                 Right[DecodingFailure, InstancePropertyValue](matchingPropType)
-              case element if element.isObject =>
```

Comment:
it coincides with last fallback case so in that sense ok to remove, but not very clear why the fallback is objectList rather than failure :thinking:
maybe keep this one and add a comment to fallback that non-objects get converted to objects too or somthing like that (or a TODO to investigate later), or a comment on why it is actually the correct fallback

Reply:
Hmm, yes that sounds like it should stay to be honest, I'm not sure what I had in mind when removing it,  but I don't feel sure of myself now...